tooling/scripts/sanity-checks.groovy

```diff
@@ -68,10 +68,11 @@ int compareVersion(String a, String b) {
     List verB = b.tokenize('.')
 
     def commonIndices = Math.min(verA.size(), verB.size())
+    def versionRegex = ~/\-.*/
 
     for (int i = 0; i < commonIndices; ++i) {
        def numA = verA[i].toInteger()
-       def numB = verB[i].toInteger()
+       def numB = (verB[i] - versionRegex).toInteger()
```

Comment:
But wait, which version actually contains the -RC qualifier? Is it the current version or some of the since versions? Both of those sounds a bit strange.

Reply:
Only `currentVersion` contains `-RC`